res/blink.js

```diff
@@ -11,12 +11,10 @@
   var sock = new WebSocket(ws);
 
   function msg(t, m) {
-    if (m === undefined) {
-      m = t;
-    } else {
-      m.type = t;
-    }
-    sock.send(JSON.stringify(m));
+    sock.send(JSON.stringify({
+      type: m === undefined ? t.type : t,
+      data: m
```

Comment:
Second case results in a message `{type: 'foo', data: { type: 'foo', bar: 'baz'}}` and the handler is called with `msg["data"]` on Julia - which is the same as it used to be. So this change is backwards compatible.